### PR TITLE
Fix some more kubejs inconsistencies by observing duplicate recipes

### PR DESCRIPTION
**Pull request by Quezler**
before:
Screen Shot 2022-05-19 at 10 06 51

after:
Screen Shot 2022-05-19 at 10 07 41

the bulk of the other recipe id overrides can be attributed to wood types being cast to another, like avocado to oak.

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ onEvent('recipes', (event) => {
         {
             output: 'industrialforegoing:fluid_collector',
             inputs: ['industrialforegoing:fluid_placer'],
-            id: 'industrialforegoing:fluid_collector'
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: was this not overriding the default fluid collector recipe?

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js [hunk 2]
```diff
@@ -465,7 +465,7 @@ onEvent('recipes', (event) => {
             sapling: 'architects_palette:twisted_sapling',
             output: 'naturesaura:shockwave_creator',
             time: 4 * time_multiplier,
-            id: 'naturesaura:shockwave_creator'
+            id: 'naturesaura:shockwave_creator' // todo: this id gets an altered_recipe_indicator, why reuse it here?
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js:468]: Perhaps @MuteTiefling knows?

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js (hunk 1 above)
Comment (Quezler) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: this base recipe already was:
https://github.com/EnigmaticaModpacks/Enigmatica6/blob/01bcfb55a9dbe7290850702c22511429cf9e19cc/kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/industrialforegoing/shaped.js#L136

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: This is a case of me forgetting to move the `base` recipe to `normal` scripts when it was changed for `expert`.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js (hunk 2 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js:468]: The default recipe is a 3x3, so the `altered_recipe_indicator` bit is valid. Looks like I just forgot to give it a proper ID of `${id_prefix}shockwave_creator`

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js (hunk 1 above)
Comment (NielsPilgaard) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: Would you mind fixing that @Quezler? 😄

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js (hunk 2 above)
Comment (NielsPilgaard) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js:468]: Would you mind fixing this too @Quezler? 😄

Comment (Quezler) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js:468]: **all** un$prefixed nature's aura recipes in this file?

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: not the entire file. Just that one recipe needs to be moved over to `kubejs/server_scripts/enigmatica/kubejs/normal/recipetypes/industrialforegoing/shaped.js`

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js (hunk 2 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js:468]: Fixed this in #4867. Just merge that into your stuff and you'll be set.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: Fixed this in #4867. Just merge that into your stuff and you'll be set.

Comment (Quezler) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: Looks like you've merged that into here, i suppose it means its all good now?

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shapeless.js:10]: ... Probably?

**Review by NielsPilgaard — APPROVED** [review]

**Pull request merged**